src/main/scala/com/raphtory/output/FileOutputFormat.scala

```diff
@@ -8,17 +8,41 @@ import org.slf4j.LoggerFactory
 import java.io.File
 
 /**
-  * # FileOutputFormat
-  * `FileOutputFormat` writes output for Raphtory Job and Partition for a pre-defined window and timestamp to File
+  * {s}`FileOutputFormat(filePath: String)`
+  *
+  *   : writes output for Raphtory Job and Partition for a pre-defined window and timestamp to File
+  *        {s}`filePath: String`
+  *          : Filepath for writing Raphtory output.
+  *   Reference: [](com.raphtory.core.algorithm.OutputFormat)
   *
   * Usage while querying or running algorithmic tests:
   *
   * ```{code-block} scala
+  * import com.raphtory.algorithms.generic.EdgeList
   * import com.raphtory.output.FileOutputFormat
-  * algorithmTest(WeightedDegree(), outputFormat, 1, 32674, 10000, List(500, 1000, 10000))
+  * import com.raphtory.core.algorithm.OutputFormat
+  * import com.raphtory.core.components.graphbuilder.GraphBuilder
+  * import com.raphtory.core.components.spout.Spout
+  * import org.apache.pulsar.client.api.Schema
+  *
+  * def setSpout(): Spout[T]
+  * def setGraphBuilder(): GraphBuilder[T]
+  * def setSchema(): Schema[T]
+  *
+  * val schema: Schema[Array[Byte]] = Schema.BYTES
+  * val spout            = setSpout()
+  * val graphBuilder     = setGraphBuilder()
+  * val graph            = Raphtory.createTypedGraph[T](spout, graphBuilder, setSchema())
   * val testDir  =  "/tmp/raphtoryTest"
-  * val defaultOutputFormat: OutputFormat  =  FileOutputFormat(testDir)
+  * val outputFormat: FileOutputFormat  =  FileOutputFormat(testDir)
+  * graph.pointQuery(EdgeList(), outputFormat, 1595303181, List())
   * ```
+  *
+  *
+  *  ```{seealso}
+  *  [](com.raphtory.core.algorithm.OutputFormat),
+  *  [](com.raphtory.core.client.RaphtoryClient)
```

Comment:
I would add `[](com.raphtory.core.client.RaphtoryGraph)` and `[](com.raphtory.core.deploy.Raphtory)` as well